mariadb-database-id/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM test-eventuate-mysql
```

Comment:
Will an image called `test-eventuate-mysql` exist in the CircleCI environment - that's just a local dev image name

Reply:
yes, the same approach was used for json tests.